### PR TITLE
Refactor/ods text

### PR DESCRIPTION
**Pull request by aesteves60**

### CODE REVIEW

File: packages/storybook/stories/components/text/2_usage.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { Meta } from '@storybook/addon-docs';
+import Usage from '@ovhcloud/ods-components/src/components/text/documentation/usage-guidelines/usage.mdx';
+
+<Meta title="ODS Components/Text/Usage Guidelines" />
```
Comment (dpellier) [inline packages/storybook/stories/components/text/2_usage.stories.mdx:4]: `<Meta title="ODS Components/Content/Text/Usage Guidelines" />`

File: packages/storybook/stories/components/text/3_migration_.from.17.x.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { Meta } from '@storybook/addon-docs';
+import Migration from '@ovhcloud/ods-components/src/components/text/documentation/migration.from.17.x.mdx';
+
+<Meta title="ODS Components/Text/Migration From 17.x" />
```
Comment (dpellier) [inline packages/storybook/stories/components/text/3_migration_.from.17.x.stories.mdx:4]: `<Meta title="ODS Components/Content/Text/Migration From 17.x" />`

File: packages/storybook/stories/components/text/4_demo.stories.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import { defineCustomElement } from '@ovhcloud/ods-components/dist/components/ods-text';
+import { ODS_TEXT_PRESETS } from '@ovhcloud/ods-components';
+import { html } from 'lit-html';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
+import type { Meta } from '@storybook/web-components';
+
+defineCustomElement();
+
+const meta: Meta = {
+  title: 'ODS Components/Text/Demo',
```
Comment (dpellier) [inline packages/storybook/stories/components/text/4_demo.stories.ts:10]: `title: 'ODS Components/Content/Text/Demo',`

File: packages/storybook/stories/components/text/4_demo.stories.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import { defineCustomElement } from '@ovhcloud/ods-components/dist/components/ods-text';
+import { ODS_TEXT_PRESETS } from '@ovhcloud/ods-components';
+import { html } from 'lit-html';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
+import type { Meta } from '@storybook/web-components';
+
+defineCustomElement();
+
+const meta: Meta = {
```
Comment (dpellier) [inline packages/storybook/stories/components/text/4_demo.stories.ts:9]: export directly? or you want to explicitly set the type?

**Review by dpellier — CHANGES_REQUESTED** [review]
Need to fix build errors

File: packages/storybook/stories/components/text/4_demo.stories.ts (hunk 4 above)
Comment (aesteves60) [reply packages/storybook/stories/components/text/4_demo.stories.ts:9]: It's just for the autocomplete & get the type for help

File: packages/ods/src/components/text/documentation/specifications/specifications-text.mdx [hunk 5]
```diff
@@ -1,7 +1,4 @@
 import { Markdown } from '@storybook/addon-docs';
 import Specs from './spec.md';
 
-## Description
```
Comment (dpellier) [inline packages/ods/src/components/text/documentation/specifications/specifications-text.mdx]: keep the title, remove just the TODO

File: packages/ods/src/components/text/documentation/migration.from.17.x.mdx [hunk 6]
```diff
@@ -6,8 +6,6 @@
 `contrasted` <img src="https://img.shields.io/badge/removed-FF0000" />
 `breakSpaces` <img src="https://img.shields.io/badge/removed-FF0000" />
 
-Has been removed.
```
Comment (dpellier) [inline packages/ods/src/components/text/documentation/migration.from.17.x.mdx]: keep `Have been removed` just fix grammar

**Review by dpellier — CHANGES_REQUESTED** [review]

File: packages/ods/src/components/text/documentation/specifications/specifications-text.mdx (hunk 5 above)
Comment (aesteves60) [reply packages/ods/src/components/text/documentation/specifications/specifications-text.mdx]: There are a duplicate information with this file: 
packages/storybook/stories/components/text/1_specifications.stories.mdx
That's why I remove it

Comment (dpellier) [reply packages/ods/src/components/text/documentation/specifications/specifications-text.mdx]: Careful there is two different part here
Capture d’écran 2024-03-18 à 17 45 15

This description is a sub part of the `# Text - Technical Specification` from the file you mentionned

**Pull request merged**